tests/pkg/mcufont/tests/01-run.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2023 Koen Zandberg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+FONT_RENDER = """
+                  █  █
+ █     █          █  █                  █████   █    ███   ███████  █
+ █     █          █  █                  █    █  █   █   █     █     █
+ █     █   ████   █  █   ████           █    █  █  █     █    █     █
+ █     █  ██  ██  █  █  ██  ██          █    █  █  █     █    █     █
+ ███████  █    █  █  █  █    █          █████   █  █     █    █     █
+ █     █  ██████  █  █  █    █          █   █   █  █     █    █     █
+ █     █  █       █  █  █    █          █    █  █  █     █    █
+ █     █  ██   █  █  █  ██  ██  █       █    █  █   █   █     █     █
+ █     █   ████   █  █   ████   █       █     █ █    ███      █     █
+                                █
+"""
+
+def testfunc(child):
```

Comment:
```suggestion

def testfunc(child):
```

To fix the flake8 complaint

Reply:
Fixed!